services/queue/docs/task-schema.md

```diff
@@ -9,9 +9,10 @@ docref:       true
 title:        "Task Schema"
 order: 100
 ---
+import SchemaTable from '../../../../src/components/SchemaTable'
```

Comment:
Is the amount of `../` correct? Also, it feels a bit weird importing something like that, although I can't formulate why. I wonder if we could have a separate small library of components and use it like a regular `taskcluster-lib-etc.`?

Reply:
> Is the amount of `../` correct?

Yes, the path is actually relative to the generated doc.

> Also, it feels a bit weird importing something like that, although I can't formulate why.

It's weird because 1 ) you probably haven't seen imports in markdown files and 2) `'../../../../` makes your eyes bleed. I admit it's not the prettiest but it really simplifies things in the UI. For example, we won't need to traverse the markdown file to insert react components like the `SchemaTable` component.